db/seeds.rb

```diff
@@ -31,7 +31,7 @@
   user.password_hash = ""
   user.email = "system@e621.net"
   user.can_upload_free = true
-  user.level = User::Levels::ADMIN
+  user.level = User::Levels::JANITOR
```

Comment:
System user, and probably the admin user, should be able to approve posts, otherwise it is likely to run into problems during execution. For the admin user, it would just be nice to have, since otherwise you have to set it manually.

Reply:
I thought it was implied by the user level but I guess not. Added